tests/utilities/fake_driver.py

```diff
@@ -21,12 +23,18 @@
 _API_KEY = "00000000-0000-0000-0000-000000000000"
 
 
-class SessionInitializationBehavior(IntEnum):
-    """Specifies whether to initialize a new session or attach to an existing session."""
+_CLOSE_BEHAVIORS = [
+    SessionInitializationBehavior.AUTO,
+    SessionInitializationBehavior.INITIALIZE_SERVER_SESSION,
+    SessionInitializationBehavior.ATTACH_TO_SESSION_THEN_CLOSE,
+]
```

Comment:
Revert this. fake_driver.py should include its own copy of the enum and only one copy of _CLOSE_BEHAVIORS.